lxd/instance/drivers/driver_qemu.go

```diff
@@ -2986,7 +2986,6 @@ func (d *qemu) addCPUMemoryConfig(cfg *[]cfgSection) (int, error) {
 	// Determine per-node memory limit.
 	memSizeMB := memSizeBytes / 1024 / 1024
 	nodeMemory := int64(memSizeMB / int64(len(hostNodes)))
-	memSizeMB = nodeMemory * int64(len(hostNodes))
```

Comment:
I can see why this looks problematic, but as its the one that is taking effect, I wonder whether its worth dropping the memSizeMB assignment above it instead. Why did you drop this line particularly?

Reply:
Oh wait, I see that both lines are doing something different now. So why are you dropping it - a reason in the commit would be useful for me to understand.

Reply:
I'll clarify in the commit message that the line was dropped because it assigns the same value as it already does two lines above.